courses/models.py

```diff
@@ -719,6 +731,13 @@ class BaseCertificate(models.Model):
 
     user = models.ForeignKey(User, null=False, on_delete=models.CASCADE)
     uuid = models.UUIDField(default=uuid.uuid4, editable=False, unique=True)
+    is_revoked = models.BooleanField(
```

Comment:
Should we rename this to `is_active` or `active` following the `live` and `is_live` convention?